man/get_region_codes.Rd

```diff
@@ -1,16 +1,301 @@
 % Generated by roxygen2: do not edit by hand
-% Please edit documentation in R/shared-methods.R
-\name{DataClass}
+% Please edit documentation in R/UK.R, R/shared-methods.R
+\name{UK}
```

Comment:
Why is the UK docs appearing in man/get_region_codes?

Reply:
I had `#' @rdname get_region_codes` in the docstring for some reason. Sorted it.